src/libstd/ffi/c_str.rs

```diff
@@ -36,8 +37,8 @@ use vec::Vec;
 ///
 /// A `CString` is created from either a byte slice or a byte vector. After
 /// being created, a `CString` predominately inherits all of its methods from
-/// the `Deref` implementation to `[libc::c_char]`. Note that the underlying
-/// array is represented as an array of `libc::c_char` as opposed to `u8`. A
+/// the `Deref` implementation to `[os::raw::c_char]`. Note that the underlying
+/// array is represented as an array of `os::raw::c_char` as opposed to `u8`. A
```

Comment:
It's probably fine to omit the `os::raw::` in these two and just mention `c_char`.